src/trumbowyg.js

```diff
@@ -266,7 +266,10 @@ jQuery.trumbowyg = {
                 link: {
                     dropdown: ['createLink', 'unlink']
                 }
-            }
+            },
+
+            blockLevelElements: ['br', 'p', 'ul', 'ol', 'table', 'img', 'address', 'article', 'aside', 'audio', 'blockquote', 'canvas', 'dl', 'fieldset', 'figcaption', 'figure', 'footer', 'form', 'h1', 'h2', 'h3', 'h4', 'h5', 'h6', 'header', 'hgroup', 'hr', 'main', 'nav', 'noscript', 'output', 'pre', 'section', 'tfoot', 'video']
```

Comment:
I think, array in settings is more convenient and reliable, but can change this to string, if you like.
P is removed in semantics code because user can proved own settings with p in it. In that case it will break parsing, because semanticCode treats p as a special element. Moreover, I think, o.blockLevelElements may be used somewhere else, so it is expected to have fullest list of possible block-level elements, including p.